modules/claims_api/app/controllers/concerns/claims_api/token_validation.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'rest-client'
+require 'mpi/responses/find_profile_response'
+require 'mpi/service'
+
+module ClaimsApi
+  module TokenValidation
+    extend ActiveSupport::Concern
+    TOKEN_REGEX = /Bearer /
+
+    included do
+      def authenticate
+        verify_access!
+      end
+
+      def verify_access!
+        verify_access_token!
+      rescue
+        render_unauthorized
+      end
+
+      # Determine if the current authenticated user is allowed access
+      # raise if current authenticated user is neither the target veteran, nor target veteran representative
+
+      def verify_access_token!
```

Comment:
looks like we're calling this twice in v2. For example: 
Claims Controller has 
`before_action :verify_access!`
Application Controller has
`before_action :authenticate`

I believe v1 is looking good though.

Reply:
Pulled it out of claims controller.